### DISCUSSION TITLE
Support for deployment on Kubernetes

### DISCUSSION THREAD

**Discussion by shaiksaiteja** [Security-Onion-Solutions/securityonion#13536] [Unsupported Versions]
Does Security Onion support Kubernetes deployment and monitoring?

**Comment by shaiksaiteja** [comment]
@reyesj2 @dougburks @defensivedepth @joeb1kenobe @christopherwoodall

**Reply by dougburks** [reply]
@shaiksaiteja From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Tagging Individuals
Please do not tag an individual in a discussion unless that individual has already volunteered to help you in that discussion.

To answer your question, Security Onion is not designed for or tested on Kubernetes.
